pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -436,6 +458,13 @@ stdenv.mkDerivation {
       echo "-L${gccForLibs}/lib/gcc/${targetPlatform.config}/${gccForLibs.version}" >> $out/nix-support/cc-ldflags
       echo "-L${gccForLibs_solib}/lib" >> $out/nix-support/cc-ldflags
     ''
+    # The above "fix" may be incorrect; gcc.cc.lib doesn't contain a
+    # `target-triple` dir but the correct fix may be to just remove the above?
+    #
+    # For clang it's not necessary (see `--gcc-toolchain` below) and for other
+    # situations adding in the above will bring in lots of other gcc libraries
+    # (i.e. sanitizer libraries, `libatomic`, `libquadmath`) besides just
+    # `libstdc++`; this may actually break clang.
```

Comment:
Which above fix is this referring to? The other edit you made in the file is almost 200 lines earlier. If it's referring to that, can we move the comment there? This block seems out of place.

Reply:
These were left after cherry picking

Reply:
I guess Ill update

Reply:
https://github.com/NixOS/nixpkgs/commit/dc6a8f9f7912363577e11520bafa040c0db14359#diff-ad7b67d4a92a2db43281c9c45aa74c32341a554fbca0b6b25be593f4648abb5eR330-R334

Reply:
@rrbutani why did you find it necessary to comment out https://github.com/NixOS/nixpkgs/commit/dc6a8f9f7912363577e11520bafa040c0db14359#diff-ad7b67d4a92a2db43281c9c45aa74c32341a554fbca0b6b25be593f4648abb5eR330?

I kept things as they were after @danielfullmer's PR